packages/web/src/components/spa/Navigation/DesktopNav/desktop_navbar.css

```diff
@@ -68,6 +68,16 @@
     align-items: center;
 }
 
+@media only screen and (max-width: 1073px) {
```

Comment:
All changes from the file desktop_navbar.css should be deleted from here. You have probably created this branch from another branch. Please next time checkout to master before creating a new branch.